docs/modules/plugins/pages/plugin-excel.adoc

```diff
@@ -74,3 +74,12 @@ Examples:
 Examples:
 {transformer=FROM_EXCEL, path=/my-excel-file.xlsx, sheet=Data, range=A2:B4}
 ----
+
+.Create xlsx file with specific content
```

Comment:
please see examples of steps docs in other plugins and provide a complete documentation